src/common/webrequest_curl.cpp

```diff
@@ -42,6 +59,12 @@
     #define CURLOPT_ACCEPT_ENCODING CURLOPT_ENCODING
 #endif
 
+#ifdef CURL_PROGRESSFUNC_CONTINUE
+    #define wxCURL_PROGRESSFUNC_CONTINUE CURL_PROGRESSFUNC_CONTINUE
+#else
+    #define wxCURL_PROGRESSFUNC_CONTINUE 0
+#endif
+
```

Comment:
```suggestion
```

This is not needed at all, we should never use `CURL_PROGRESSFUNC_CONTINUE`.

Reply:
This was removed in 3e5fd54.